components/script/lib.rs

```diff
@@ -10,7 +10,6 @@
 // building rustdoc or running clippy.
 #![cfg_attr(crown, register_tool(crown))]
 #![cfg_attr(any(doc, clippy), allow(unknown_lints))]
```

Comment:
```suggestion
```

Reply:
This was added, to not show crown warning in doc/clippy, so we can safely remove it.

Reply:
Done.